components/login/login_controller/index.ts

```diff
@@ -2,20 +2,33 @@
 // See LICENSE.txt for license information.
 
 import {connect} from 'react-redux';
-import {bindActionCreators} from 'redux';
-
+import {ActionCreatorsMapObject, bindActionCreators, Dispatch} from 'redux';
 import {getConfig, getLicense} from 'mattermost-redux/selectors/entities/general';
-import {getTeamByName, getMyTeamMember} from 'mattermost-redux/selectors/entities/teams';
+import {getMyTeamMember, getTeamByName} from 'mattermost-redux/selectors/entities/teams';
 import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
 import {RequestStatus} from 'mattermost-redux/constants';
+import {GlobalState} from 'mattermost-redux/types/store';
+import {ActionFunc, GenericAction} from 'mattermost-redux/types/actions';
+import {ServerError} from 'mattermost-redux/types/errors';
+import {Team} from 'mattermost-redux/types/teams';
 
 import {addUserToTeamFromInvite} from 'actions/team_actions';
-
 import {login} from 'actions/views/login';
 
-import LoginController from './login_controller.jsx';
+import LoginController from './login_controller';
+
+interface State extends GlobalState {
```

Comment:
We have a type already for the webapp version of the redux state, it's in `types/store`.
However, looks like it's missing the `initialized` field. Can you add it in that type definition instead of using it here?
Thanks for catching that!